drawBot/context/baseContext.py

```diff
@@ -537,7 +537,34 @@ def transform(self, transformMatrix, center=(0, 0)):
         aT.setTransformStruct_(transformMatrix[:])
         self._path.transformUsingAffineTransform_(aT)
 
-    # boolean operations
```

Comment:
This comment should stay, but below your new methods.

Reply:
Oops, I will fix that.